<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetPas</class>
 <widget class="QWidget" name="ForgetPas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>313</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能家居-使徒行者</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>231</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="UserNameLabel">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="UserNameLineEdit"/>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="PhoneLabel">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>手机号</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PhoneLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="ConfirmPushButton">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>验证</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="ConfirmLabel">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>验证码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ConfirmPhoneLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="SendPushButton">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="CancelPushButton">
        <property name="font">
         <font>
          <family>华文楷体</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
